src/plugins/administrative_templates/ui/i18n/filteroptions_en.ts

```diff
@@ -0,0 +1,130 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS>
+<TS version="2.1" language="en_US">
+<context>
+    <name>FilterOptionsDialog</name>
+    <message>
+        <location filename="../filteroptionsdialog.ui" line="14"/>
+        <source>Dialog</source>
+        <translation>Dialog</translation>
+    </message>
+</context>
+<context>
+    <name>FilterOptionsWidget</name>
+    <message>
+        <location filename="../filteroptionswidget.ui" line="14"/>
+        <source>Form</source>
+        <translation>Form</translation>
+    </message>
+    <message>
+        <location filename="../filteroptionswidget.ui" line="50"/>
+        <source>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select options below to enable and change or disabletypes of global filters that will be applied to the Administrative Templates nodes.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</source>
```

Comment:
It will be cleaner if we disable rich text here.

Reply:
fixed